salt/etc-hosts/hosts.jinja

```diff
@@ -12,7 +12,7 @@
 ### admin nodes ###
 {%- set admins = salt['mine.get']('roles:admin', 'network.interfaces', 'grain') %}
 {%- for admin_id, ifaces in admins.items() %}
-{{ salt.caasp_net.get_primary_ip(host=admin_id, ifaces=ifaces) }} {{ salt.caasp_net.get_nodename(host=admin_id) }} {{ salt.caasp_net.get_nodename(host=admin_id) }}.{{ pillar['internal_infra_domain'] }} {{ admin_id }} {{ admin_id }}.{{ pillar['internal_infra_domain'] }}
+{{ salt.caasp_net.get_primary_ip(host=admin_id, ifaces=ifaces) }} {{ salt.caasp_net.get_nodename(host=admin_id) }} {{ salt.caasp_net.get_nodename(host=admin_id) }}.{{ pillar['internal_infra_domain'] }} {{ admin_id }} {{ admin_id }}.{{ pillar['internal_infra_domain'] }} ldap.{{ pillar['internal_infra_domain'] }}
```

Comment:
Adding the same entry several times won't have the desired effect. Instead of this, I think we want a separate line outside of this loop with something like: `127.0.0.1 ldap.{{ pillar['internal_infra_domain'] }}`.

Reply:
And only in the admin role, since the rest of the nodes won't have ldap running locally.

Reply:
updated